app/models/user.rb

```diff
@@ -20,4 +20,19 @@ class User < ApplicationRecord
       transition all - [:closed] => :closed
     end
   end
-end
\ No newline at end of file
+
+  def taunt_police
+    pp "taunt police"
+  end
+
+  def taunt_police2
+    pp "taunt police"
+  end
+
+  def taunt_police3
```

Comment:
```suggestion
    pp 'taunt police'
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

Reply:
something wrong...
suggestion location がずれる